bot_cli/constants.py

```diff
@@ -4,27 +4,26 @@
 COMMANDS: dict = {
     "help": "shows available commands",
     "hello": "prints 'How can I help you?'",
-    "add-contact <user> <phone>": "adds a new contact with phone",
-    "add-birthday <user> <birthday>": "adds birthday to a contact in format [DD.MM.YYYY]",
-    "add-phone <user> <phone>": "adds a phone number to a contact",
-    "add-email <user> <email>": "adds email to a contact",
-    "add-address <user> <address>": "adds address to a contact",
-    "add-note <note>": "adds a new note",
+    "add-contact {user} {phone}": "adds a new contact with phone",
```

Comment:
I specifically wanted them to be `<>` and added a screenshot and a link http://docopt.org/ documneting why to my task.

Reply:
Reverted.
Note: If you changed the format of variable in the strings - it should be updated everyehere and not only in 1 place.